File: backend/src/main/java/com/safetunnel/backend/vpn/server/VpnServerRepository.java
package com.safetunnel.backend.vpn.server;

import com.safetunnel.backend.users.VpnServer;
import org.springframework.data.jpa.repository.JpaRepository;
import java.util.UUID;

public interface VpnServerRepository extends JpaRepository<VpnServer, UUID> {
    VpnServer findByHost(String host);
}

File: backend/src/main/java/com/safetunnel/backend/vpn/server/VpnServerRepository.kt
package com.safetunnel.backend.vpn.server

import com.safetunnel.backend.users.VpnServer
import org.springframework.data.jpa.repository.JpaRepository
import java.util.UUID

interface VpnServerRepository : JpaRepository<VpnServer, UUID> {
    fun findByHost(host: String): VpnServer?
}